### ISSUE TITLE
demo and test

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1427]
Where are they hosted? They are both quite slow. Not such a problem for test, but maybe the demo should  be on oliver 0 if it isn't already?

**Comment by kimrutherford** [comment]
The test and demo Cantos are hosted on the new machine (oliver0) just like the main Canto.  They are allocated fewer resources so they don't interfere with the main Canto.  I haven't had a look at how resources are distributed in a while.  I'll have a look today.

**Comment by kimrutherford** [comment]
I've increased the number of server processes for demo and test.  That might help a little bit.

I also noticed that Chado query caching was disabled for demo and test.  Querying Chado to get the existing annotation is rather slow so for the main Canto we have been caching the results.  So the first time you go to a session the existing annotation is slow to appear, but should be fast after the first load.

I've turned the caching on for test and demo.  That will only change the loading speed of existing (Chado) annotation though.
